docker/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM alpine:3.6
+
+COPY restic /usr/bin
```

Comment:
I'm sorry. You're right.

```
+ /usr/local/bin/docker build --rm=true -f docker/Dockerfile -t 7529baea7f9e7b304ab8641ab2b86f08286e8279 . --pull=true
Sending build context to Docker daemon 60.63 MB
...

Step 2/4 : COPY restic /usr/bin
 ---> a7b7e770bc93
```

So, the build's context = current working directory = Git repo location.